### DISCUSSION TITLE
Az El pointer

### DISCUSSION THREAD

**Discussion by ckies22** [Stellarium/stellarium#3151] [Q&A]
Hello All,

I'm keen to make a simple Az El pointer with stepper motors (controlled by Arduino) and to then point it to the selected object by simply sending the azimuth and elevation values via Serial port. I've looked at the telescope control options, but these all are quite complex relative to what I'm keen to do. I do not need super accuracy, nor accurate tracking. And I'm not too keen on using the Ra/Dec system, as it introduces additional complexities. I'm also new to Stellarium and scripting, so forgive me if this is a stupid question :)

Thanks in advance for any ideas or hints!

**Comment by gzotti** [comment]
Check out https://www.researchgate.net/publication/356126347_Stellector_a_laser_aided_night_sky_explorer_for_teaching_astronomy
